src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ICompilationExtensions.cs

```diff
@@ -206,5 +206,11 @@ public static ImmutableArray<IAssemblySymbol> GetReferencedAssemblySymbols(this
 
         public static INamedTypeSymbol? DisallowNullAttribute(this Compilation compilation)
             => compilation.GetTypeByMetadataName(typeof(DisallowNullAttribute).FullName!);
+
+        public static INamedTypeSymbol? DataMemberAttribute(this Compilation compilation)
```

Comment:
```suggestion
        public static INamedTypeSymbol? SystemRuntimeSerializationDataMemberAttribute(this Compilation compilation)
```

Reply:
➡️ Did not change this since the other types in the file follow the current pattern